css/components/_heading.scss

```diff
@@ -0,0 +1,4 @@
+.plugin-header {
```

Comment:
If the main element of the file is called `plugin-header`, the file should be called `_plugin-header.scss`